Autopilot/AttitudeManager/StateMachine.c

```diff
@@ -24,14 +23,10 @@ int downlinkP1Timer = 0;
 int downlinkP2Timer = 0;
 int imuTimer = 0;
 char AMUpdate = 0;
+char flightUpdate = 0;
```

Comment:
should be static. Also rename to `updateFlight` and add comments describing what this variabe does